Assets/Scripts/Interactables/Interactable.cs

```diff
@@ -27,6 +30,14 @@ protected virtual void Start()
     audioSource = GetComponent<AudioSource>();
   }
 
+  protected virtual void LateUpdate()
+  {
+    if (interactionCount == maxInteractionCount)
```

Comment:
would this be resolved automatically if the interaction tip was a child of the Interactable?
if so, when we instantiate it, we could just call `SetParent` and set it to the `Interactable`'s transform

Reply:
@gabxgonz good idea! 

After removing the `interactionCount`, I went with an `isDestroyed` flag instead, so it is now: 


```
  protected virtual void LateUpdate()
  {
    if (isDestroyed)
    {
      interactableUI.DestroyInteractionTip();
    }
  }
```

I have also updated `InteractableUI` to set the interactable game object as the parent: 
`interactionTipInstance.gameObject.transform.SetParent(gameObject.transform);`

This caused some issues with the positioning, but since we haven't hammered out the UI design for the tip, I just set it to render on the overlay screen space for now.

Reply:
maybe we can pivot to screen space anyway in the design phase and just light up the relevant buttons. zelda games often keep the controls on screen the whole time